client/src/components/Header/Header.jsx

```diff
@@ -16,16 +16,34 @@ import {
 } from '@mui/material';
 import MenuIcon from '@mui/icons-material/Menu';
 import HowToRegIcon from '@mui/icons-material/HowToReg';
-import LogoutIcon from '@mui/icons-material/Logout';
+import LoginIcon from '@mui/icons-material/Login';
 import ArrowBackIcon from '@mui/icons-material/ArrowBack';
 import './Header.scss';
-import { routes } from '../../routes';
+import routes from '../../routes';
 
 export default function Header({ darkMode, handleThemeChange }) {
   const [showMenu, setShowMenu] = useState(false);
   const { pathname } = useLocation();
   const navigate = useNavigate();
 
+  const [deferredPrompt, setDeferredPrompt] = useState(null);
+  window.addEventListener('beforeinstallprompt', (e) => {
+    e.preventDefault();
+    setDeferredPrompt(e);
+  });
+
+  const install = async () => {
+    deferredPrompt.prompt();
+  };
+
+  const installButton = () => {
+    return (
+      <Button variant="contained" sx={{ mx: 4, my: 2 }} onClick={install}>
+        Zainstaluj
+      </Button>
+    );
+  };
+
```

Comment:
I think Szymon deliberately removed that from a codebase so we should not add it back, am I right @szymonkwak ?

Reply:
yes, you are right :)